src/lib/components/InputTriboxdate.svelte

```diff
@@ -201,14 +209,14 @@
 		<!-- Use date field (hidden) to take advantage of browser validator api -->
 		<input
 			type="date"
-			id={component.id}
+			id="{component.id}-input"
```

Comment:
We are not using "-input" as a key to locating / identifying a component are we?  as what would happen if I called a component "postcode-input" - this would create a <div id="postcode-input" and a <input id="postcode-input-input"  - thusly we cannot use -input as a search term to locate input controls.

I know you have probably thought about this, just stating.